templates/default/sql/dml/2.0.0.sql.erb

```diff
@@ -1,3 +1,5 @@
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("rmyarn_user", "<%= node['hops']['rm']['user'] %>");
 
 REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("spark_executor_min_memory", 1024);
+
+REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("check_nodemanagers_status", false);
```

Comment:
the value should come from an variable defined in attributes/default.rb and also defined in metadata.rb. Default value is "false"